src/test/resources/com/scriptbasic/testprograms/TestStringFunctions.bas

```diff
@@ -1,3 +1,24 @@
+' Test ltrim, trim, rtrim functions
+v = 1234
+assert("ltrim_with_num", ltrim(v)="1234")
+assert("trim_with_num", trim(v)="1234")
+assert("rtrim_with_num", rtrim(v)="1234")
+
+v = " 1234 "
+assert("ltrim", ltrim(v)="1234 ")
+assert("trim", trim(v)="1234")
+assert("rtrim", rtrim(v)=" 1234")
+
+' Test null values
+assert("ltrim_with_null", IsNull(ltrim(xxx)))
```

Comment:
instead of `xxx` you could use `undef`. More expressive.

Reply:
I renamed variable name.